src/app/resources/sesion.py

```diff
@@ -0,0 +1,35 @@
+from flask_restful import Resource
+from flask import request
+import jwt
+
+from app.models.usuario import Usuario
+
+class SesionResource(Resource):
+    def post(self):
+        post_data = request.get_json()
+        # Buscar el usuario por email
+        usuario = Usuario.query.filter_by(
+            email=post_data.get('email')
+        ).first()
+        if usuario and usuario.verificar_password(post_data.get('password')):
+            auth_token = usuario.generar_auth_token()
+            return {'auth_token': auth_token.decode()}, 200
+        return {'mensaje': 'Email o constraseña invalidos'}, 400
+
+    def get(self):
+        auth_header = request.headers.get('Authorization')
+        auth_token = ''
+        if auth_header:
+            # el auth_token esta despues Bearer
+            split = auth_header.split(" ")
+            if len(split) == 2 and split[0] == 'Bearer':
+                auth_token = split[1]
+        if auth_token:
+            try:
+                usuario_id = Usuario.validar_auth_token(auth_token)
+                return {'usuario_id': usuario_id}, 200
+            except (jwt.ExpiredSignatureError, jwt.InvalidTokenError):
+                # token invalido o caducado
+                return {}, 403
```

Comment:
Por qué 403? No debería ser un Forbidden debería ser un Not Authorized igual que en el otro caso

Reply:
Ee vi en un par de lados que usaban esta convención pero supongo que podemos poner en ambos el mismo codigo